proposals/lip-0066.md

```diff
@@ -45,6 +45,14 @@ We define the following constants:
 | `PrivateKeyEd25519` | bytes  | Byte sequences of length `ED25519_PRIVATE_KEY_LENGTH`. | An Ed25519 private key. |
 | `PrivateKeyBLS`     | bytes  | Byte sequences of length `BLS_PRIVATE_KEY_LENGTH`. | An BLS private key. |
 
+## uint32be Function
+We define the function `uint32be` to convert bytes to unsigned integer values.
+
+```python
+def uint32be(chainID: bytes) -> int:
```

Comment:
I just realize I made a mistake with my previous comment: the function `uint32be` we define in other LIPs converts from uint32 to bytes, here instead we want to convert from bytes to uint32, hence the previous name you had was fine, sorry about that.

Reply:
Replaced with the old one.